pydy/tests/test_bodies.py

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env python
+
+SYMPY_VERSION = sm.__version__
+
+
+class TestBodies():
+
+    def test_body_init(self):
+        body = Body('body')
+
+        assert body.name is None
+        assert body.mass == Symbol('mass')
+
+        sys = System(method='joints')
+        sys.add_body(body)
+
+        assert body.name == 'body_1'
+
+        body_length = Length(body.name)
+        assert body_length.symbol == Symbol(body.name + "_length")
```

Comment:
Think some about a Parameter class though. We have an infinite number of parameters and they all have a symbol and potentially a value. So there could possible be need for that, but it could just be a tuple or a namedtuple. I'm not sure it is that valuable to have a Parameter class but I could be convinced if you have a utility in mind.